src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/health/VersionOutputter.kt

```diff
@@ -1,21 +0,0 @@
-package uk.gov.justice.digital.hmpps.hmppstemplatepackagename.health
-
-import org.slf4j.LoggerFactory
-import org.springframework.boot.context.event.ApplicationReadyEvent
-import org.springframework.boot.info.BuildProperties
-import org.springframework.context.annotation.Configuration
-import org.springframework.context.event.EventListener
-
-@Configuration
-class VersionOutputter(buildProperties: BuildProperties) {
-  private val version = buildProperties.version
-
-  @EventListener(ApplicationReadyEvent::class)
-  fun logVersionOnStartup() {
-    log.info("Version {} started", version)
```

Comment:
Unnecessary.  This was added when spring boot didn't output it, but these days I can see
```
Starting HmppsTemplateKotlinKt v2024-06-26.4379.35dae93 using Java 21.0.3 with PID 1 (/app/app.jar started by appuser in /app)
```
Also the `AppVersion` is included in every single log entry.